### PR TITLE
fix: activationEvents list commands has reverted

### PR DESCRIPTION
**Pull request by xhulz**
change the `*` from original commands

refer to: https://github.com/trufflesuite/vscode-ext/issues/166

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -36,7 +36,29 @@
...
+    "onCommand:truffle-vscode.createProject",
+    "onCommand:truffle-vscode.disconnectProject",
+    "onCommand:truffle-vscode.debugTransaction",
+    "onCommand:truffle-vscode.signInToInfuraAccount",
+    "onCommand:truffle-vscode.signOutOfInfuraAccount",
+    "onCommand:truffle-vscode.startGanacheServer",
+    "onCommand:truffle-vscode.stopGanacheServer",
+    "onCommand:truffle-vscode.views.explorer.refreshExplorer",
+    "onCommand:truffle-vscode.views.deployments.refresh",
+    "onDebug"
```
Comment (acuarica) [inline package.json:61]: Should we reintroduce the full list of events[1]? These are the events that are missing

```json
    "onCommand:truffle-vscode.createContract",
    "onCommand:truffle-vscode.views.dashboard.startDashboardServer",
    "onCommand:truffle-vscode.views.dashboard.stopDashboardServer",
    "onCommand:truffle-vscode.views.dashboard.restartDashboardServer",
    "onCommand:truffle-vscode.views.dashboard.openDashboard",
    "onCommand:truffle-vscode.views.dashboard.copyRPCEndpointAddress",
```

---

https://github.com/trufflesuite/vscode-ext/pull/136/files

Comment (xhulz) [reply package.json:61]: Hey @acuarica, Thank you :)

I'm gonna put those ones

Comment (xhulz) [reply package.json]: @acuarica done!

**Review by acuarica — APPROVED** [review]
Thanks!! lgtm

**Review by kevinbluer — APPROVED** [review]
👍

**Review by michaeljohnbennett — APPROVED** [review]

**Pull request merged**